Scheduler/DynamicNodeGroups/DynamicNodeGroups/Program.cs

```diff
@@ -109,7 +109,7 @@ static async Task Main(string[] args)
                     int newAllocationCount = job.AllocatedNodes.Count;
                     Console.WriteLine("newAllocationCount: {0}", newAllocationCount);
                     Console.WriteLine("allocationCount: {0}", allocationCount);
-                    
+
```

Comment:
In some other files, the using block was changed to using declaration. Is there a particular reason why the using statement on line 21 is still in block form?

Reply:
Changed to using declaration. Visual Studio suggests using declaration.